control-plane/msp-operator/src/main.rs

```diff
@@ -356,11 +356,12 @@ impl ResourceContext {
         self.ctx.http.pools_api()
     }
 
-    fn block_devices_api(
-        &self,
-    ) -> &dyn openapi::apis::block_devices_api::tower::client::BlockDevices {
-        self.ctx.http.block_devices_api()
-    }
+    // TODO: Uncomment the below
```

Comment:
FYI: Changes in this file would be reverted in the next PR, where we would add the BlockDevice Ops over grpc.

Reply:
Done.